### PR TITLE
Sentence'ify some labels/strings

### PR DESCRIPTION
**Pull request by kingthorin**
- AbstractMultipleOptionsBaseTablepanel.java > Change "Remove Without COnfirmation" to be a question "Remove without confirmation?" and have the checkbox following instead of leading.
- Messages.properties > Add periods to "Do not show this message again." labels/strings.

Related to zaproxy/zaproxy#2000

### CODE REVIEW

**Review by psiinon — APPROVED** [review]

**Review by ricekot — APPROVED** [review]
Thanks!

File: zap/src/main/java/org/zaproxy/zap/view/AbstractMultipleOptionsBaseTablePanel.java [hunk 1]
```diff
@@ -226,6 +227,7 @@ protected String getRemoveWithoutConfirmationLabel() {
     protected final JCheckBox getRemoveWithoutConfirmationCheckBox() {
         if (removeWithoutConfirmationCheckBox == null) {
             removeWithoutConfirmationCheckBox = new JCheckBox(getRemoveWithoutConfirmationLabel());
+            removeWithoutConfirmationCheckBox.setHorizontalTextPosition(SwingConstants.LEFT);
```
Comment (thc202) [inline zap/src/main/java/org/zaproxy/zap/view/AbstractMultipleOptionsBaseTablePanel.java:230]: Why this change and why a question now?
